apps/src/lib/tools/jsdebugger/MercatorMap.jsx

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {geoMercator, geoPath} from 'd3-geo';
+import {feature} from 'topojson-client';
+
+const PROJECTION_RADIUS = 70;
+const MAP_WIDTH = 400;
+const MAP_HEIGHT = 400;
+
+class MercatorMap extends React.Component {
+  static propTypes = {
+    data: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
+  };
+
+  state = {
+    mercator: '',
+    data: this.props.data
```

Comment:
Is there a reason to copy the data from props to state? I think it should be accessible from props the same as it would be from state.